Code/Framework/AzCore/AzCore/IO/Streamer/Scheduler.cpp

```diff
@@ -35,6 +37,10 @@ namespace AZ::IO
         m_threadData.m_streamStack = AZStd::move(streamStack);
     }
 
+    Scheduler::~Scheduler()
+    {
+    }
```

Comment:
The the destructor can be declared `= default` in the header.

Reply:
I'll have to keep in cpp file,to keep the FileRequest.h inclusion out of the header
Otherwise the `using FileRequestPtr = AZStd::intrusive_ptr<ExternalFileRequest>` needs to be full specified for every inclusion of StreamerContext.h